app/decorators/manageiq/providers/storage_manager_decorator.rb

```diff
@@ -29,8 +29,8 @@ def quadicon
         :tooltip => n_("%{number} Cloud Volume", "%{number} Cloud Volumes", t) % {:number => t}
       }
       icon[:top_right] = {
-        :text    => t = cloud_volume_snapshots.size,
-        :tooltip => n_("%{number} Cloud Volume Snapshot", "%{number} Cloud Volume Snapshots", t) % {:number => t}
+        :text    => t = storage_resources.size,
```

Comment:
Wouldn't this break the other storage managers?

Reply:
no. it does not. If a a storage manager has no storage resources it will display 0.
![cinder-quad-icon](https://user-images.githubusercontent.com/41486601/93183464-b0d77080-f743-11ea-8d5f-5e46c5f702a5.jpg)

Reply:
Well yes, but it changes the behavior of the quadicon. Instead of showing the snapshot, now you want to show something else and so you break the original functionality...

Reply:
We discussed this with our storage team and it was all have agreed that it will be more helpful to display the number of pools/backends which is what StorageResource represents instead of volume snapshots. This is of course not critical but rather a matter of preference.

Reply:
You can put this into a more specific decorator that would not affect the other storage quadicons, you just need to put into the right folder and name it properly. I guess it should be `ManageIQ::Providers::AutoSDE::StorageManager` or something like that.

Reply:
moved to a relevant folder for the autosde provider